### PR TITLE
fixes a syntax error in a string

### PR DESCRIPTION
**Pull request by approachcircle**
# IMPORTANT
this pr will fix all issues with the openfl backend, provided that [this](https://github.com/MasterEric/openfl) fork of openfl is used  (default branch), instead of the base repository

should close issues #102, #97 and #91 if merged

### CODE REVIEW

**Comment by beeanie** [comment]
Thank you sooo much

**Comment by beeanie** [comment]
i will add this to my fork

File: polymod/Polymod.hx [hunk 1]
```diff
@@ -480,7 +480,7 @@ class Polymod
 					if (apiScore < PolymodConfig.apiVersionMatch)
 					{
 						Polymod.error(VERSION_CONFLICT_API,
-							'Mod "$origDir" was built for incompatible API version ${meta.apiVersion)}, current version is "${apiVersion}"', SCAN);
+							'Mod "$origDir" was built for incompatible API version ${meta.apiVersion}, current version is "${apiVersion}"', SCAN);
```
Comment (MasterEric) [inline polymod/Polymod.hx:483]: This fix looks correct. I do not know why this built on my machine without this change (maybe I fixed the issue and forgot to commit it myself?).

**Review by MasterEric — CHANGES_REQUESTED** [review]

File: polymod/backends/OpenFLBackend.hx [hunk 2]
```diff
@@ -11,23 +11,28 @@ class OpenFLBackend extends StubBackend
 }
 #else
 #if !nme
+
+import openfl.utils.AssetCache;
+
 class OpenFLBackend extends LimeBackend
 {
+	var assetCache = new AssetCache();
```
[the remark this answers is not in the document]
Comment (approachcircle) [reply polymod/backends/OpenFLBackend.hx:19]: what actually is the other fix that you've mentioned?

[the remark this answers is not in the document]
Comment (MasterEric) [reply polymod/backends/OpenFLBackend.hx:19]: The one where you removed the `)` from the string in this same PR

[the remark this answers is not in the document]
Comment (approachcircle) [reply polymod/backends/OpenFLBackend.hx:19]: i now understand that the program should compile using your fix, but for some reason, my haxe compiler still complains that it cannot find the functions you added in AssetCache.hx

**Comment by approachcircle** [comment]
actually, on second thought, closing this PR completely probably isn't the right thing to do as one of the commits is still beneficial, so i'll reopen it and revert the commits which are not needed

**Comment by MasterEric** [comment]
Thanks, I'll accept that change.

Since this appears to be an issue for people I'm suspending work on 1.5 to make a 1.4.3 to try to fix this.

**Pull request merged**